src/clj/rems/db/test_data.clj

```diff
@@ -445,7 +445,7 @@
 
 (defn- create-catalogue-item! [resource workflow form localizations]
   (let [id (:id (db/create-catalogue-item!
-                 {:title "non-localized title" :resid resource :wfid workflow :form form}))]
+                 {:title "non-localized title" :resid resource :wfid workflow :form form :state "enabled"}))]
```

Comment:
Isn't `enabled` now the default?

Reply:
Yes, but it is on lower levet, in queries.sql

Reply:
But doesn't `db/create-catalogue-item!` here refer to the db fn?